.gitignore

```diff
@@ -33,7 +33,6 @@ dist/
 /node_modules/
```

Comment:
Do we still need this? and docs?

Reply:
We still use npm for `build-emails`, so probably `node_modules` must stay. But in case of `docs` for me, it looks like we can remove it. @maarcingebala, what do you think?

Reply:
Yes, this probably needs to stay for now since we still have some packages in package.json.